src/scripts/settings/appSettings.js

```diff
@@ -132,6 +132,18 @@ class AppSettings {
         return toBoolean(this.get('limitSupportedVideoResolution'), false);
     }
 
+    /**
+     * Get or set 'Preferred Transcode Audio' state.
+     * @param {string|undefined} val - Flag to set 'Preferred Transcode Audio' or undefined.
+     * @return {string} 'Preferred Transcode Audio' state.
```

Comment:
"state" is more for flags (checkboxes).
```suggestion
     * Get or set preferred transcode audio.
     * @param {string|undefined} val - Preferred transcode audio or undefined.
     * @return {string} Preferred transcode audio.
```